content/markdown/research-distribution-center/historical-imagery-archive.md

```diff
@@ -60,38 +64,43 @@ abstract: Our archive consists of over 1,000,000 frames with dates as far back a
+      </ul>
+  </div>
+  <div class="col-md-6 acquiring-agency"><h3>United States Air Force</h3>
+      <a href="https://tnris-org-static.s3.amazonaws.com/images/usaf_full.jpg" data-toggle="lightbox" data-gallery="example-gallery"data-title="United States Air Force (USAF)">
+          <img class="thumbnail pull-left" src="https://tnris-org-static.s3.amazonaws.com/images/airforce_historical_th.jpg">
+      </a>
+      <ul class="list-clean">
+          <li><strong>Media Type:</strong> Black and White prints</li>
+          <li><strong>General Scale:</strong> 1:56000</li>
+          <li>TNRIS has a small collection of images from the United States Air Force (USAF) spanning the mid 1950s to late 1960s.</li>
+          <li><strong>About the USAF:</strong><br>The Air Force Historical Research Agency, established in 1943, was designated custodian of documents, narratives, reports, orders, and photographs for the Air Force in times of war and peace.  The AFHR serves as a resource for scholars, researchers, government agencies, and the general public.</li>
+      </ul>
+  </div>
 </div>
-<div class="acquiring-agency">
-    <h3>United States Department of Agriculture and Agricultural Stabilization and Conservation Service</h3>
-    <a href="https://tnris-org-static.s3.amazonaws.com/images/usda_acs_full.jpg" data-toggle="lightbox" data-gallery="example-gallery" data-title="United States Department of Agriculture (USDA) and Agricultural Stabilization and Conservation Service (ASCS)">
-        <img class="thumbnail pull-left" src="https://tnris-org-static.s3.amazonaws.com/images/usda_acs_th.jpg">
-    </a>
-    <ul class="list-clean">
-        <li><strong>Media Type:</strong> Black and White, Color Infrared, Natural Color prints.  Line Indexes also available for most counties</li>
-        <li><strong>General Scale:</strong> 1:2000 (ASCS/USDA), 1:60000(NHAP), 1:40000(NAPP)</li>
-        <li>In addition to the United States Department of Agriculture and Agricultural Stabilization and Conservation Service (USDA/ASCS) collections, TNRIS also houses collections from the National High Altitude Program (NHAP) 1980-1989 and the National Aerial Photography Program (NAPP) 1987-2003 both being USGS interagency coordinated programs.</li>
-        <li><strong>About the USDA:</strong><br>The USDA is home to one of the country’s largest aerial film libraries housing more than 10 million images.  Images date back from 1955 to present with coverage of most of the United States and its territories.</li>
-    </ul>
+<div class="row">
+  <div class="col-md-6 acquiring-agency">
+      <h3>United States Department of Agriculture and Agricultural Stabilization and Conservation Service</h3>
+      <a href="https://tnris-org-static.s3.amazonaws.com/images/usda_historical_full.jpg" data-toggle="lightbox" data-gallery="example-gallery" data-title="United States Department of Agriculture (USDA) and Agricultural Stabilization and Conservation Service (ASCS)">
```

Comment:
the lightbox image url is incorrect here. that image file does not exist in the s3 bucket.